src/pages/registration/files_and_license_tab/FilesTableRow.tsx

```diff
@@ -138,29 +137,24 @@ export const FilesTableRow = ({ file, removeFile, baseFieldName, showFileVersion
         <TableCell sx={{ minWidth: isArchived ? '5.5rem' : '' }}>{prettyBytes(file.size)}</TableCell>
 
         <TableCell>
-          <Field name={administrativeAgreementFieldName}>
+          <Field name={fileTypeFieldName}>
             {({ field }: FieldProps) => (
-              <Tooltip title={t('registration.files_and_license.administrative_contract')}>
-                <Checkbox
-                  {...field}
-                  data-testid={dataTestId.registrationWizard.files.administrativeAgreement}
-                  checked={field.value}
-                  disabled={disabled}
-                  inputProps={{
-                    'aria-labelledby': administrativeAgreementId,
-                  }}
-                  onChange={(event) => {
-                    const newAssociatedFileType: AssociatedFileType =
-                      field.value === true ? 'UnpublishedFile' : 'UnpublishableFile';
-                    setFieldValue(fileTypeFieldName, newAssociatedFileType);
-
-                    field.onChange(event);
-                    setFieldValue(publisherVersionFieldName, null);
-                    setFieldValue(licenseFieldName, null);
-                    setFieldValue(embargoFieldName, null);
-                  }}
-                />
-              </Tooltip>
+              <Checkbox
+                {...field}
+                data-testid={dataTestId.registrationWizard.files.toPublishCheckbox}
+                checked={field.value === FileType.UnpublishedFile}
+                disabled={disabled}
+                inputProps={{
+                  'aria-labelledby': markForPublishId,
+                }}
+                onChange={(event, checked) => {
+                  if (!checked) {
+                    setFieldValue(fileTypeFieldName, FileType.UnpublishableFile);
+                  } else {
+                    setFieldValue(fileTypeFieldName, FileType.UnpublishedFile);
+                  }
+                }}
+              />
```

Comment:
Diskuterte litt med Kjetil her i lokalet - burde dette kanskje være en knapp med tilhørende confirm-dialog? Vet jeg er sen til festen her, og mulig dette har vært diskutert. Men det føles litt unaturlig at dette er en sjekkboks. Det er en litt drastisk endring til å være en sjekkboks. Og burde denne kanskje vært i siste kolonne? Og er ikke sjekkboksen invertert? "Merk for publisering" med avhuket boks høres ut som filen er publisert? En knapp i siste kolonne (uten header) med tekst "publiser" / "avpubliser" / "arkiver" hadde vært ryddig?

Dette er bare tanker jeg får når jeg tester. Trenger ikke gjøre noe med dette, da implementasjonen er i tråd med design :)